tlatools/org.lamport.tlatools/test/tlc2/tool/liveness/ModelCheckerTestCase.java

```diff
@@ -111,7 +111,7 @@ public void setUp() {
 
 		try {
 			// TEST_MODEL is where TLC should look for user defined .tla files
-			ToolIO.setUserDir(BASE_DIR + TEST_MODEL + path);
```

Comment:
Why doesn't this break all existing tests?

Reply:
`BASE_PATH` was already defined in this class as `BASE_DIR + TEST_MODEL` - someone must have missed this in refactoring